PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+## Description
```

Comment:
I'm not sure we have to be as prescribed for this PR template. Perhaps a note at the top "Please be sure your PR addresses the following:" can help make it more open, but still provide good direction.